### PR TITLE
Closes #1817: Show newlines in textboxes

### LINKED ISSUE #1817: Display of textbox metadata fields should not ignore new lines entered by a user

### PR DESCRIPTION
**Pull request by wtr-icm**

### CODE REVIEW

File: dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java [hunk 1]
```diff
@@ -430,14 +430,15 @@ private String getFieldDisplayValue(String value, DatasetFieldType fieldType) {
             if (StringUtils.isBlank(format)) {
                 format = "#VALUE";
             }
-            String sanitizedValue = !fieldType.isSanitizeHtml() ?
-                    value :
-                    MarkupChecker.sanitizeBasicHTML(value);
+            if (FieldType.TEXTBOX.equals(fieldType.getFieldType())) {
+                value = value.replaceAll("\\n", "<br>");
```
Comment (madryk) [inline dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java:434]: What will happen if we enter text "\n" into description?

Comment (wtr-icm) [reply dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java:434]: I've checked and nothing erroneous happens.

Comment (madryk) [reply dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java:434]: When we will display such field that will contain text `\n` - we will display text `\n` or the extra line will be produced?

Comment (wtr-icm) [reply dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java:434]: The text ``\n`` would be displayed as ``\n`` (two characters).
The newline is implicitly converted to ``\n`` (one character) and will be displayed as newline.

**Review by madryk — APPROVED** [review]

**Pull request merged**